### PR TITLE
Move elements up and down

### PR DESCRIPTION
**Pull request by annatraussnig**
### Up/down inline UI

Screenshot 2021-11-30 at 09 52 02

Supported elements: 

- [x]  infobox
- [x]  picture
- [x]  picture group
- [x]  list/enum
- [x]  chart
- [x]  dynamic component
- [x]  typgraphic quote
- [x]  long quote
- [x]  embed

Arrows appears on focus. Jumps through breakout elements.

### DC/Charts edition

Screenshot 2021-11-30 at 09 51 47

For dynamic components and charts, the arrows and the edit UI were merged into a single UI, with the help of a new context.

### CODE REVIEW

**Comment by felixmichel** [comment]
also works for audio and tweets

**Comment by felixmichel** [comment]
I can move things around all the way, meaning that I am also able to move something past the article end and then to the top of the page again. Is this intentional?

Bildschirmfoto 2021-12-01 um 13 49 40

**Review by felixmichel — COMMENTED** [review]
works fine. It would be nice if we could keep the focus on the elment that we move up and down in order to be able to make this action fast and easily multiple times.

**Comment by annatraussnig** [comment]
The article collection feature is a bit tricky and I would like to leave it out for the current release. However, I added the scroll feature you suggested @felixmichel, makes the whole experience quite better indeed!

**Comment by felixmichel** [comment]
nice! I like it!

File: components/editor/modules/infobox/index.js [hunk 1]
```diff
@@ -86,20 +88,27 @@ export default ({ rule, subModules, TYPE }) => {
...
-        renderNode({ node, children, attributes }) {
+        renderNode({ node, children, attributes, editor }) {
           if (!serializerRule.match(node)) return
 
           const hasFigure =
             figureModule && node.nodes.find(n => n.type === figureModule.TYPE)
+
           return (
             <Container
+              key='content'
```
Comment (tpreusse) [inline components/editor/modules/infobox/index.js:99]: Seems unnecessary

**Review by tpreusse — APPROVED** [review]
LGTM

List within info boxes are ignored / can not be moved right?

**Comment by annatraussnig** [comment]
Lists in info boxes isn't something I was aware of. I'll check what's going on in that front.

**Comment by annatraussnig** [comment]
Re lists inception: lists in infoboxes were indeed movable but should have been handled quite differently from top level elements. I decided against support for the time being, as you suggested too. If this proves a recurring problem for our Produktators, we will handle this case post hoc.

**Comment by tpreusse** [comment]
Awesome. Thank you for 🚢 ing it.

One thought re. the new `isInInfobox` logic: We could also invert this—`InlineUI.js` could check if it's a child of center and only show the buttons in that case. And if it later supported moving within others expand it there. Anyhow not important just a thought if we expand on it or have to revise the code later on.

**Comment by annatraussnig** [comment]
good point, too late though, mouhahahahahahahahahahahahaha!

**Pull request merged**